### PR TITLE
[6.15.z] Fix AK tests assertion errors (#16220)

### LINKED ISSUE #16262: [Failed-AutoCherryPick] - Fix AK tests assertion errors
#### Auto-Cherry-Pick WorkFlow Failure:
- To Branch: 6.15.z
- [Failed Cherrypick Action](https://github.com/SatelliteQE/robottelo/actions/runs/10739693914)
- [Parent Pull Request](https://github.com/SatelliteQE/robottelo/pull/16220)

### PR DESCRIPTION
**Pull request by vsedmik**
Manual cherry-pick of #16220 

Closes #16262

### CODE REVIEW

**Comment by Satellite-QE** [comment]
**PRT Result**
```
Build Number: 8522
Build Status: UNSTABLE
PRT Comment: pytest tests/foreman/cli/test_activationkey.py -k test_positive_create_content_and_check_enabled or test_positive_add_custom_product --external-logging
Test Result : ========= 2 failed, 145 deselected, 178 warnings in 676.87s (0:11:16) ==========
```

**Comment by Satellite-QE** [comment]
**PRT Result**
```
Build Number: 8523
Build Status: UNSTABLE
PRT Comment: pytest tests/foreman/cli/test_activationkey.py -k test_positive_create_content_and_check_enabled or test_positive_add_custom_product --external-logging
Test Result : ==== 1 failed, 1 passed, 145 deselected, 178 warnings in 731.68s (0:12:11) =====
```

**Comment by Satellite-QE** [comment]
**PRT Result**
```
Build Number: 8527
Build Status: SUCCESS
PRT Comment: pytest tests/foreman/cli/test_activationkey.py -k test_positive_create_content_and_check_enabled or test_positive_add_custom_product --external-logging
Test Result : ========= 2 passed, 145 deselected, 184 warnings in 732.27s (0:12:12) ==========
```

File: tests/foreman/cli/test_activationkey.py [hunk 1]
```diff
@@ -204,9 +202,13 @@ def test_positive_create_content_and_check_enabled(module_org, module_target_sat
...
     content = module_target_sat.cli.ActivationKey.product_content(
-        {'id': result['activationkey-id'], 'organization-id': module_org.id}
+        {
+            'id': result['activationkey-id'],
+            'organization-id': module_org.id,
+            'content-access-mode-all': 1,
+        }
     )
-    assert content[0]['default-enabled?'] == 'true'
+    assert content[0]['default-enabled?'] == 'false'
```
Comment (vsedmik) [inline tests/foreman/cli/test_activationkey.py:211]: Custom content is disabled by default in 6.15, see https://docs.redhat.com/en/documentation/red_hat_satellite/6.15/html-single/managing_content/index#Restricting_Hosts_Access_to_Content_content-management

**Comment by Gauravtalreja1** [comment]
trigger: test-robottelo
pytest: tests/foreman/cli/test_activationkey.py -k 'test_positive_create_content_and_check_enabled or test_positive_add_custom_product'

**Comment by Satellite-QE** [comment]
**PRT Result**
```
Build Number: 8541
Build Status: SUCCESS
PRT Comment: pytest tests/foreman/cli/test_activationkey.py -k test_positive_create_content_and_check_enabled or test_positive_add_custom_product --external-logging
Test Result : ========= 2 passed, 145 deselected, 178 warnings in 687.96s (0:11:27) ==========
```

**Pull request merged**